components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -329,12 +329,66 @@ export function prebuildStatusIcon(prebuild?: PrebuildWithStatus) {
     }
 }
 
+function PrebuildStatusDescription(props: { prebuild: PrebuildWithStatus }) {
+    switch (props.prebuild.status) {
+        case "queued":
+            return <span>Prebuild is queued and will be processed when there is execution capacity.</span>;
```

Comment:
**thought**: Sounds accurate but a bit scary for humans, no? Maybe we could use something like the following:

> Prebuild is queued and will be processed in a moment.